### PR TITLE
Adding XboxPlaydates quote app after tests

### PR DESCRIPTION
**Pull request by PalmettoBling**
# Description
Adding in the XboxPlaydates app.  The app displays a random quote pulled via the Xbox Playdates API from the Xbox Playdates quote database, which is made up of community submitted quotes of things said during our community streams.  The App displays the quote number that can be referenced, the quote itself and who said the quote.

xboxplaydates

# Copilot

### Summary
🎮📝🧠

This pull request adds a new applet called XboxPlaydates, which shows a random quote from the Xbox Playdates website on the Tidbyt device. It includes a manifest file and a Starlark code file for the applet.

> _`XboxPlaydates` app_
> _Starlark code and manifest_
> _Winter fun on Tidbyt_

### Walkthrough
* Create and populate a manifest file for the XboxPlaydates applet ([link](https://github.com/tidbyt/community/pull/1751/files?diff=unified&w=0#diff-18a1460d9d3be437ba2926f6097359b926d7d517c73cc47542ee845c51e54d05R1-R8))
* Implement the applet logic and UI in Starlark ([link](https://github.com/tidbyt/community/pull/1751/files?diff=unified&w=0#diff-3c1c58311e0e3182aec290ca57216979d690627993de4a13cc0407a5360a5adcR1-R65))

### CODE REVIEW

**Comment by PalmettoBling** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by PalmettoBling** [comment]
For the record of using the http.star:
image
The Xbox Playdates API is a Node / Express website/Twitch bot I have running in Azure with the documents being retrieved stored in an Azure Cosmos Database.  I utilize the app and API regularly for other functions.  The API is used for a Google Chrome / MS Edge extension, discord bots, website usage, Twitch chatbots, ect. and is resilient.

File: apps/xboxplaydates/xboxplaydates.star [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                ),
+            ],
+        ),
+    )
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
```
Comment (matslina) [inline apps/xboxplaydates/xboxplaydates.star:58]: This fields seems to be unused.

**Review by matslina — CHANGES_REQUESTED** [review]

File: apps/xboxplaydates/xboxplaydates.star (hunk 1 above)
Comment (PalmettoBling) [reply apps/xboxplaydates/xboxplaydates.star]: You are correct.  Thank you for your time and review.  Removed the function and library call.  Artifact left from app file generation process.

**Review by PalmettoBling — COMMENTED** [review]
Updated file to remove function and library call.  Committed to branch.

**Review by matslina — APPROVED** [review]

**Pull request merged**